.github/workflows/linter.yml

```diff
@@ -90,3 +90,29 @@ jobs:
 
     - name: Run prettier
       run: find . -name "*.js" | grep -v "subprojects" | xargs npx prettier  --print-width 120 --check
+
+  python:
```

Comment:
I think i missed this `changes` job. What is it?

Reply:
It detect which files has been modified and set some variables.

In this case `needs.changes.outputs.python`

Reply:
So I can create a conditional run for the python linter (only run if a python files has been modified)